MC/config/analysis_testing/json/analysis-testing-data.json

```diff
@@ -1,4 +1,50 @@
 {
+  "EMCClusterMonitorTask": {
```

Comment:
@benedikt-voelkel I formatted the config json as it is the only way to have a meaningful comparison wrt to the one produced by the DPL workflow and to check if it's correctly up 2 date

Reply:
That's fine and I think good to do so every now and then. So thanks!